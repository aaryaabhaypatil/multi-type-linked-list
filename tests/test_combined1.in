NEW 3
we
are
university
NEW 4
A
s
d
f
View al
VIEW all
VIEW ALL
REMOVE 2
REMOVE 0
VIEW -1
VIEW 0
TYPE 0
TYPE 1
