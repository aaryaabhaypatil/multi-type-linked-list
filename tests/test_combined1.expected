List 0: we -> are -> university
List 1: A -> s -> d -> f
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
Number of lists: 2
List 0
List 1
INVALID COMMAND: REMOVE
List 0 has been removed.

Number of lists: 1
List 1
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
INVALID COMMAND: INPUT
char -> char -> char -> char

